viite-backend/viite-main/src/test/scala/fi/liikennevirasto/viite/dao/ProjectLinkDAOSpec.scala

```diff
@@ -368,6 +370,91 @@ class ProjectLinkDAOSpec extends FunSuite with Matchers {
     }
   }
 
+  test("Test batchUpdateProjectLinksToReset to update new values") {
```

Comment:
Kuvaavampi nimi, esim "Test batchUpdateProjectLinksToReset When batch updating project links back to original values Then all the project link values should reset." tms